### ISSUE TITLE
Dustjs output limited to 65536 characters

### ISSUE THREAD

**Issue by tomalexander** [linkedin/dustjs#801]
To reproduce: write a template that outputs more than 65536 characters. Your output will be truncated.

**Comment by ansumanswain** [comment]
could you attach the respective repo link

**Comment by tomalexander** [comment]
Yep, reproduction at https://github.com/tomalexander/dustjs_reproduce_65536
